test/spmd/tensor/test_pointwise_ops.py

```diff
@@ -90,6 +90,35 @@ def test_dropout_errors(self):
                 torch.nn.functional.dropout,
             )
 
+    @with_comms
+    def test_mul(self):
```

Comment:
maybe this test can mainly testing the `aten.mm.out` (`torch.mul(...out=)`)? you can fix the CI by deleting the xfail in the `test_dtensor_ops.py`, which testing torch.mul, but we still need to test out variant.